zerver/openapi/zulip.yaml

```diff
@@ -18721,7 +18721,7 @@ components:
           description: |
             Whether the user has muted the stream. Muted streams do
             not count towards your total unread count and do not show up in
-            `All messages` view (previously known as `Home` view).
+            `Message feed` view (previously known as `ALl messages` view).
 
```

Comment:
What's up with the capitalization here?

Reply:
Thank you for noticing it, I've fixed it.